vendor/modules.txt

```diff
@@ -1937,12 +1937,10 @@ k8s.io/component-base/configz
 k8s.io/component-base/featuregate
 k8s.io/component-base/featuregate/testing
 k8s.io/component-base/logs
-k8s.io/component-base/logs/datapol
```

Comment:
Note: we still need the [data policy tags for static analysis](https://github.com/kubernetes/kubernetes/blob/1d52f442ee745f38a190e0b0563405b6b30a3750/hack/testdata/levee/levee-config.yaml#L25-L32) so I left them alone, but I believe this isn't being vendored directly into k8s anymore with the removal of log sanitization.

Reply:
Keeping the data policy tags in, so they could be used to benefit static analysis, was the primary concern raised during the SIG Security meeting. Thank you for calling this out explicitly.